pytest_splunk_addon/standard_lib/requirement_tests/test_templates.py

```diff
@@ -214,4 +223,5 @@ def test_requirement_params(
             f" Field_extraction_check: {field_extraction_check} \n"
             f" Field extraction errors: {json.dumps(missing_key_value, indent=4)} \n"
             f" sourcetype of ingested event: {sourcetype} \n"
+            f" sourcetype in requirement file event: {sourcetype_req_file} \n"
```

Comment:
If `transport_type` is not in the list of what we support, this line does not work.

Reply:
We actually have a default if none is present in the requirement file. https://github.com/splunk/pytest-splunk-addon/blob/db7a059fa8e129c9cbc5a4a74c1aedc699b7dad9/pytest_splunk_addon/standard_lib/requirement_tests/test_generator.py#L284

Reply:
Hm, okay, this code (https://github.com/splunk/addonfactory-workflow-requirement-files-unit-tests/blob/80cccd0353431c5358b9ec5545d527b92a93d20f/test_lib/test_transport_attrib.py#L101) will raise an error if there is no sourcetype I believe.

Reply:
I just have a strong feeling that I was getting an error if I did not provide a sourcetype / host / source.

Reply:
Fixed by adding default assignment. Thank you